arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/ctx/ArexThreadLocal.java

```diff
@@ -137,7 +138,11 @@ private static void restoreValues(HashMap<ArexThreadLocal<Object>, Object> backu
         public static void setTtlTo(HashMap<ArexThreadLocal<Object>, Object> ttlValues) {
             for (Map.Entry<ArexThreadLocal<Object>, Object> entry : ttlValues.entrySet()) {
                 ArexThreadLocal<Object> threadLocal = entry.getKey();
-                threadLocal.set(entry.getValue());
+                Object value = entry.getValue();
+                if (value instanceof CallDepth) {
+                    continue;
+                }
+                threadLocal.set(value);
```

Comment:
Suggest filtering in method `io.arex.agent.bootstrap.ctx.ArexThreadLocal.Transmitter#captureValues`